### DISCUSSION TITLE
Source Formatting

### DISCUSSION THREAD

**Discussion by Romulus10** [fennelLabs/Fennel-Protocol#50] [Announcements]
Before you commit new changes, make sure you run the setup script again (to get any new Rust updates and make sure you're on the right Rust branch) and run `cargo fmt` on the project. I've made a couple stylistic changes to the `rustfmt.toml` and double-checked that it runs, that should help us keep everything clean and consistent without too much extra trouble.

**Comment by isaacadams** [comment]
update: from a discussion during our last dev meeting, we decided to add `cargo fmt` as a check for every PR, where it will fail if the files are not formatted properly
